setup.py

```diff
@@ -121,13 +121,14 @@ def read(*names, **kwargs):
         'open source', 'scan', 'license', 'package', 'dependency',
         'copyright', 'filetype', 'author', 'extract', 'licensing',
     ],
-    python_requires='>=2.7,<3',    
+    python_requires='>=2.7,<4',    
```

Comment:
As I said I cannot merge this for now as develop does not support Python3 yet. So do not send a PR with this for now. It will have to either stay local or it will need to be defined conditionally based on the current Python version (a simple sys check)